WordPress/src/test/java/org/wordpress/android/ui/mysite/MySiteSourceManagerTest.kt

```diff
@@ -0,0 +1,264 @@
+import org.wordpress.android.test
+import org.wordpress.android.ui.mysite.MySiteSource.MySiteRefreshSource
+import org.wordpress.android.ui.mysite.MySiteUiState.PartialState.SelectedSite
+import org.wordpress.android.ui.mysite.cards.domainregistration.DomainRegistrationSource
+import org.wordpress.android.ui.mysite.cards.post.PostCardsSource
+import org.wordpress.android.ui.mysite.cards.quickstart.QuickStartCardSource
+import org.wordpress.android.ui.mysite.dynamiccards.DynamicCardMenuViewModel.DynamicCardMenuInteraction
+import org.wordpress.android.ui.mysite.dynamiccards.DynamicCardsSource
+import org.wordpress.android.util.analytics.AnalyticsTrackerWrapper
+import org.wordpress.android.util.config.MySiteDashboardPhase2FeatureConfig
+
+@ExperimentalCoroutinesApi
+@Suppress("LargeClass")
+@RunWith(MockitoJUnitRunner::class)
+class MySiteSourceManagerTest : BaseUnitTest() {
+    @Mock lateinit var analyticsTrackerWrapper: AnalyticsTrackerWrapper
+    @Mock lateinit var domainRegistrationSource: DomainRegistrationSource
+    @Mock lateinit var scanAndBackupSource: ScanAndBackupSource
+    @Mock lateinit var currentAvatarSource: CurrentAvatarSource
+    @Mock lateinit var dynamicCardsSource: DynamicCardsSource
+    @Mock lateinit var postCardsSource: PostCardsSource
+    @Mock lateinit var quickStartCardSource: QuickStartCardSource
+    @Mock lateinit var siteIconProgressSource: SiteIconProgressSource
+    @Mock lateinit var selectedSiteSource: SelectedSiteSource
+    @Mock lateinit var mySiteDashboardPhase2FeatureConfig: MySiteDashboardPhase2FeatureConfig
+    private lateinit var mySiteSourceManager: MySiteSourceManager
+    private val selectedSite = MediatorLiveData<SelectedSite>()
+    private lateinit var allRefreshedMySiteSources: List<MySiteSource<*>>
+    private lateinit var selectRefreshedMySiteSources: List<MySiteSource<*>>
+
+    @InternalCoroutinesApi
+    @Suppress("LongMethod")
+    @Before
+    fun setUp() = test {
+        selectedSite.value = null
+        mySiteSourceManager = MySiteSourceManager(
+                analyticsTrackerWrapper,
+                currentAvatarSource,
+                domainRegistrationSource,
+                dynamicCardsSource,
+                quickStartCardSource,
+                scanAndBackupSource,
+                selectedSiteSource,
+                postCardsSource,
+                siteIconProgressSource,
+                mySiteDashboardPhase2FeatureConfig)
+
+        allRefreshedMySiteSources = listOf(
+                selectedSiteSource,
+                siteIconProgressSource,
+                quickStartCardSource,
+                currentAvatarSource,
+                domainRegistrationSource,
+                scanAndBackupSource,
+                dynamicCardsSource,
+                postCardsSource
+        )
+
+        selectRefreshedMySiteSources = listOf(
+                quickStartCardSource,
+                currentAvatarSource
+        )
+    }
+
+    /* ON REFRESH */
```

Comment:
Minor (🔍): Consider living a space above and below the grouping comments.

PS: This is found elsewhere as well.